src/consensus/blockchain.py

```diff
@@ -46,17 +47,19 @@ class ReceiveBlockResult(Enum):
     DISCONNECTED_BLOCK = 5  # Block's parent (previous pointer) is not in this blockchain
 
 
-class Blockchain:
+class Blockchain(BlockchainInterface):
+    SUB_BLOCKS_CACHE_SIZE = 2000
```

Comment:
This is not large enough, we need at least a whole epoch worth of sub blocks, plus a buffer. We can use the constants